### ISSUE TITLE
Strange black border on bubble popus cards

### ISSUE THREAD

**Issue by ex3tera** [Clooos/Bubble-Card#396]
Hi folks,
yesterday I updated HA system and all HACS integrations and I discovered a strange black border on cards (custom button cards) on the bubble pop-ups. Moreover I think the transparency of the popups is changed. Does anyone experienced the same issue?

Current HA version:
<img src="https://github.com/Clooos/Bubble-Card/assets/62616219/a639176c-3fbf-420f-b0af-4e478c435ab3" width="500">

Before updates:
<img src="https://github.com/Clooos/Bubble-Card/assets/62616219/2fff2d59-1203-496f-99de-43c28618f020" width="200">

After updates:
<img src="https://github.com/Clooos/Bubble-Card/assets/62616219/d4077d2a-ff48-4d9a-9f36-7e32f749d785" width="200">

UPDATE1:
Ive done some test and seems related with "Bubble card" version higher than 1.6.4.
- 1.6.4 -> working fine
- 1.7 - transparency of popups is lost
- 1.71 or higher -> black border on cards

I think however that its not only "bubble card" related but more the interation between custom button card updates and bubble card updates.

**Comment by Termite254** [comment]
Transperency of popus is lost since 1.7 for me too. I do not have the black border problem.

**Comment by Clooos** [comment]
Hi and thanks for the feedback! I'm aware of this issue and I'm working on it!

**Comment by szamfirov** [comment]
Same issue with transparency for me too. I tried multiple settings but couldn't manage to get the transparency back. I had to revert back to 1.6.4 for things to work again.

Something strange which I noticed, if I set `bg_color`, `bg_blur` and `bg_opacity` options (and remove all styles) and refresh the page multiple times, I could see the pop-up being transparent every once in a while but 99% of the time it's not 🤔
